pkg/controllers/membercluster/v1beta1/membercluster_controller.go

```diff
@@ -65,11 +65,11 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	klog.V(2).InfoS("Reconcile", "memberCluster", req.NamespacedName)
 	var mc clusterv1beta1.MemberCluster
-	mcObjRef := klog.KObj(&mc)
 	if err := r.Client.Get(ctx, req.NamespacedName, &mc); err != nil {
 		klog.ErrorS(err, "failed to get member cluster", "memberCluster", req.Name)
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
+	mcObjRef := klog.KObj(&mc)
```

Comment:
v1beta1 is supposed to be copied from v1alpha1, does this mean the v1alpha1 has this nil ref too?